lib/conjur/api/authn.rb

```diff
@@ -63,10 +63,38 @@ def login username, password, account: Conjur.configuration.account
       # @param [Hash] params Additional params to send to authenticator
       # @return [String] A JSON formatted authentication token.
       def authenticator_authenticate authenticator, service_id, account: Conjur.configuration.account, options: {}
+        JSON.parse authenticator_authenticate_post authenticator, service_id, account: account, options: options
```

Comment:
Should the default behavior of `authenticator_authenticate` be to use the POST method?

If yes - should the GET routing be removed entirely?

Reply:
> Should the default behavior of authenticator_authenticate be to use the POST method?

Yeah, probably. Optimally, we would use the endpoint that is credential-agnostic for code and refresh token exchanges.

Through our progress so far on the refresh token feature branch, we've maintained the GET behavior for backwards-compatibility. Maybe we should consider maintaining it here, too, until we officially remove it.